src/DIRAC/Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -74,8 +74,9 @@
 
 # BEWARE: this import makes it impossible to instantiate this CE client side
 from DIRAC.WorkloadManagementSystem.DB.PilotAgentsDB import PilotAgentsDB
+from DIRAC.WorkloadManagementSystem.Client import PilotStatus
```

Comment:
There is not states map in HTCondorCE?

Reply:
The states map is defined in `/Resources/Computing/BatchSystems/Condor`.